web/templates/stream_settings/copy_email_address_modal.hbs

```diff
@@ -1,6 +1,9 @@
 <div class="copy-email-modal">
+    {{> ../dropdown_widget_with_label
+      widget_name="sender_channel_email_address"
+      label=(t 'Who should be the sender of the Zulip message sent to this channel?')}}
     <p class="question-which-parts">
-        {{t "Which parts of the email should be included in the Zulip message sent to this channel?"}}
+        {{t "Which parts of the email should be included in the message?"}}
```

Comment:
I think we want to make "message" plural here, since it's a reusable email address.